apiary.apib

```diff
@@ -16,43 +16,76 @@ current session status.
 
 ### Sign In [PUT]
 
-+ Request (application/json)
++ Request (application/vnd.api+json)
+
+    + Headers
+
+            Accept: application/vnd.api+json
+    
+    + Body
+
+            {
+                "data": {
+                    "attributes": {
+                        "username": "john@example.com",
+                        "password": "secret"
+                    }
+                }
+            }
+
+
++ Response 201 (application/vnd.api+json)
 
         {
-            "username": "john@example.com",
-            "password": "secret"
+            "links": {
+                "self": "https://example.com/session"
+            },        
+            "data": {
+                "id": "sessionid123"
+            }
```

Comment:
I wonder if we could include the profile to the response of the sign in?

For example, let's say I want to greet the user after sign in. To avoid sending an aditional request to `/session/account/profile`, I wonder if we could do something like `POST /session?include=account.profile.name`? I couldn't tell from the specs if the `include` query is only for GET requests or not, any thoughts?